api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java

```diff
@@ -43,6 +47,12 @@
 
     private ContextualData contextualData;
 
+    public CurrentAuthenticatorEnrollment currentAuthenticator;
```

Comment:
I think we could expose this differently, if you look at usages of this, we only use one very specific element for webauthn. I think we could create a new nullable webauthn specific object to use for this instead.

Reply:
refactored around this again.